static/js/background_image_loader.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+  "use strict";
+
+  // Define palette of fallback colors
+  const COLORS = [
```

Comment:
I think it's okay to clump these hexes together instead of listing it. I also don't think we need to note what the colors are, as long as it's pulling a different color when reloading and when the image doesn't load.